Applications/CMakeLists.txt

```diff
@@ -11,6 +11,7 @@ add_subdirectory(HexEditor)
 add_subdirectory(IRCClient)
 add_subdirectory(KeyboardMapper)
 add_subdirectory(KeyboardSettings)
+add_subdirectory(Loader)
```

Comment:
The dynamic loader is not really an application :)
Let's put this somewhere else. `Userland/DynamicLoader/` perhaps?